middleware/packet-forward-middleware/docs/integration.md

```diff
@@ -0,0 +1,156 @@
+# Integration
+This document provides instructions on integrating and configuring the Packet Forward Middleware (PFM) within your
```

Comment:
Usually useful to give folks a prediction for how long it will take them, in hour experience. 

Is this 5 minutes or more like 5 hours? Doesn't need to be precise but should ballpark

Reply:
I can definitely get that information included here as well.